web/pgadmin/tools/sqleditor/static/js/components/QueryToolDataGrid/Editors.jsx

```diff
@@ -178,8 +226,8 @@ export function TextEditor({row, column, onRowChange, onClose}) {
 
   const value = row[column.key] ?? '';
   const [localVal, setLocalVal] = React.useState(value);
-  const {getCellElement} = useContext(RowInfoContext);
   const pgAdmin = usePgAdmin();
+  const editorRef = React.useRef(null);
```

Comment:
editorRef doesn't looks to be used anywhere. check other places.

Reply:
It is being passed to Resizable div and then used there.

Reply:
it is passed to ref prop which is not required. Essentially, forwardRef would also be not required unless you want to use StyledDiv in your JSON editor.

Reply:
Yes I have already removed that.

Reply:
You edited your original comment to which I replied.

Reply:
Yes, after replying I realized forward ref is not required so when I updated the code I removed it.